frontend/src/libs/enums/components/components.ts

```diff
@@ -0,0 +1,2 @@
+export { NotificationPosition } from './notification-position.enum.js';
+export { NotificationTheme } from './notification-theme.enum.js';
```

Comment:
As these to enums are specific for notification component - they should be grouped by this component purpose so here in `libs/enums/components` create separate `notification` folder and place them through that folder and then reexport them here like
```suggestion
export { NotificationPosition, NotificationTheme } from './notification/notification.js';
```